test/suite/test_chunkcache02.py

```diff
@@ -98,6 +98,9 @@ def test_chunkcache02(self):
         self.session.checkpoint()
         self.session.checkpoint('flush_tier=(enabled)')
 
+        # Assert the new chunks are ingested. 
+        self.assertGreater(self.get_stat(wiredtiger.stat.conn.chunk_cache_newly_inserted), 0)
```

Comment:
Upon reconsideration, I'm thinking of renaming this stat - `chunk_cache_newly_inserted`. The term `newly_inserted` in FTDC might suggest that these chunks were recently inserted. However, that may not be the case, as chunks can be inserted even after objects have been ingested. 

@ruby-oujo,  what do you think of  `chunk_cache_chunks_ingested`?